invenio_queues/ext.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, app, connection_pool, entry_point_group):
 
     @cached_property
     def queues(self):
-        # NOTE: import iter_entry_points here so it can be mocked in tests
```

Comment:
the `NOTE` is used for translations and would be picked up by it (in the future, since now there are no translations)